### PR TITLE
GH1266 Slightly revise version wording to be congruous with example

### PR DESCRIPTION
**Pull request by shrielenee**
## This PR is a:
[ ] New topic
[X] Content fix or rewrite
[ ] Bug fix or improvement
 

## Summary

Revised wording of versioning in Conceptual overview section to be congruous with example given. Performed basic copyedit. This PR satisfies the request in GitHub issue #1266. 
 
When this pull request is merged, it will ensure the conceptual example and versions mentioned are congruous.
 
------------------
Thank you for your contribution!
 
Before submitting this pull request, please make sure you have read our Contribution Guidelines and your PR meets our contribution standards:
https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing_docs.html
 
Please fill out as much information as you can about your PR to help speed up the review process.
If your PR addresses an existing GitHub Issue, please refer to it in the title or Additional Information section to make the connection.
 
We may ask you for changes in your PR in order to meet the standards set in our Contribution Guidelines. PR's that do not comply with our guidelines may be closed at the maintainers' discretion.

Feel free to remove this section before creating this PR.
-->

### CODE REVIEW

File: guides/v2.1/migration/migration-overview-how.md [hunk 1]
```diff
@@ -1,52 +1,39 @@
...
+* **Map** - set of rules that describe connections between Magento 1.x and Magento 2.x data structures
+* **Mode** - represented by a separate Data Migration Tool command; defines the basic mode of operation as:
+  * *Settings* - migrates the system configuration and website-related settings
+  * *Data* - migrates database assets in bulk
+  * *Delta* - migrates incremental changes, such as orders and inventory
 
 ## Migration modes (phases)
 
-This guide provides information about migration using the Data Migration Tool that can transfer and adapt data from Magento 1 to Magento 2 stores. The migration in overall is split into three phases (or "modes"):
+This guide provides information about migration using the Data Migration Tool that can transfer and adapt data from Magento 1 to Magento 2 stores. The migration in overall is split into three phases (or modes):
```
Comment (bdenham) [inline guides/v2.1/migration/migration-overview-how.md]: Simplify and clarify. Maybe something like: "The data migration tool transfers and adapts data from Magento 1 to Magento 2 stores. The tool works by splitting the migration into three phases (or modes):"

File: guides/v2.1/migration/migration-overview-how.md [hunk 2]
```diff
@@ -1,52 +1,39 @@
...
 
 *	Configuration settings
-
 *	Data
-
 *	Delta (changes since the last run)
 
-Each mode is divided into steps, each of which is responsible for transferring particular data (for example, {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %} rewrite step, EAV step, settings step, and so on). At the begin of a run, a step checks the table structures of Magento 1 and Magento 2 for consistency. Then the actual data is transferred to Magento 2. In the end, this data is verified.
+Each mode is divided into steps, each of which is responsible for transferring particular data, such as the {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %}, the rewrite step, the EAV step, and the settings step. At the beginning of a run, a step checks the table structures of Magento 1 and Magento 2 for consistency. Then the actual data is transferred to Magento 2 and is verified.
```
Comment (bdenham) [inline guides/v2.1/migration/migration-overview-how.md]: More clarification needed. Maybe something like: "The data migration tool divides each mode into steps that define the data migrated. Examples include the URL rewrite step, the EAV step, and the settings step. Each step verifies that the Magento 1 and Magento 2 table structures are consistent. If they are, the migration tool transfers and verifies the data for that step."

File: guides/v2.1/migration/migration-overview-how.md [hunk 3]
```diff
@@ -1,52 +1,39 @@
...
+The data migration tool recognizes the differences in database structure between Magento 1.x and 2.x versions. Most of these database structural differences are declared in map files. Each step in the process uses map files to transform data for use in your Magento 2 store.
 
-For example, when you transform data from a {{site.data.var.ce}} 1.8.0.0 database to {{site.data.var.ce}} 2.x.x, the map file accounts for the fact that a table was renamed and renames it accordingly in the destination database. If there are no differences in data structure or data format, the Data Migration Tool transfers it as-is to the Magento 2 database, including data from tables created by extensions.
+For example, when you transform data from a {{site.data.var.ce}} 1.8.0.0 database to {{site.data.var.ce}} 2.x.x, the map file accounts for the fact that a table was renamed, and renames it accordingly in the destination database. If there are no differences in data structure or data format, the Data Migration Tool transfers it as-is, including data from tables created by extensions, to the Magento 2 database.
```
Comment (bdenham) [inline guides/v2.1/migration/migration-overview-how.md]: Don't know if we should capitalize **Data Migration Tool** or not. Not sure if that's an internal tool's title, but not capitalizing is probably cool.

File: guides/v2.1/migration/migration-overview-how.md (hunk 1 above)
Comment (shrielenee) [reply guides/v2.1/migration/migration-overview-how.md]: 👍

File: guides/v2.1/migration/migration-overview-how.md (hunk 2 above)
Comment (shrielenee) [reply guides/v2.1/migration/migration-overview-how.md]: 👍

File: guides/v2.1/migration/migration-overview-how.md (hunk 3 above)
Comment (shrielenee) [reply guides/v2.1/migration/migration-overview-how.md]: I think it is a specific tool title. But that's a good question!

**Pull request merged**